### DISCUSSION TITLE
Does the simulation results match real world data?

### DISCUSSION THREAD

**Discussion by tdzdog** [MartinHahner/LiDAR_fog_sim#4] [Q&A] [answered]
Thanks for your excellent work. I am hoping to use the simulation to generate some synthetic foggy data, e.g. on KITTI. Hence, I am wondering how much does the simulated data match with real-world foggy data? But I found only some minor qualitive discussion is represented in the paper, no quantitive discussion.

**Comment by MartinHahner** [comment, marked as answer]
Hi and thank you for your question. 
It is difficult to measure realism. Not just for point clouds, but also e.g. for computer-generated images (see [here](https://diglib.eg.org/xmlui/bitstream/handle/10.2312/EGWR.EGWR01.235-248/235-248.pdf?sequence=1&isAllowed=y)). 
If you can tell us how we could quantitatively measure the realism, please tell us. We could not find any meaningful metric.
Greets, Martin
